src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandler.java

```diff
@@ -120,10 +121,11 @@ protected boolean shouldIncreaseIndent() {
 
     /**
      * The function checks if the new keyword is a child of chained method calls,
-     * it checks if the new is directly followed by equal operator or return operator.
+     * it checks if the new is directly followed by equal operator, ternary operator or
+     * return operator.
      *
      * @return true if the new it is chained method calls and new keyword is directly followed
-     *         by assign or return
+     *         by assign, ternary or return
```

Comment:
Main summary doesn't mention `assign`. Looks like it should be added?

Reply:
i didnt get this, could u please explain a little more?

Reply:
In main summary equal operator is mentioned, i think equal operator should not come in chained method call.
should i replaced equal with assign?

Reply:
should i mention `equal` in return description as well?

Reply:
Where is equal coming from? I am not seeing it in the code as one of the TokenTypes.

Reply:
not in my branch i have removed it already but u can see it in checkstyle/master.
https://github.com/checkstyle/checkstyle/blob/df5cda9b438f8430420d51807c18d1ce51ac489a/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandler.java#L123

Reply:
You can search on git history and see if there is a reason why `equal` was added. If you find `equal` serves no purpose, then you can remove it altogether.